### DISCUSSION TITLE
Why React native tools showing multiple `React Native Packager` for multiple RN folder in workspace?

### DISCUSSION THREAD

**Discussion by flixyudh** [microsoft/vscode-react-native#1879] [General]
I have 3 RN Folder and put them all in one workspace, but i don't know why RNT show 3 RN Packager
<img width="656" alt="Screen Shot 2022-11-30 at 13 42 12" src="https://user-images.githubusercontent.com/19633881/204726021-72e9e5a7-6dc1-4c2c-80bf-7385550187ce.png">

when i click one of them, vscode ask me which folder should i chose
<img width="501" alt="Screen Shot 2022-11-30 at 13 45 20" src="https://user-images.githubusercontent.com/19633881/204726763-a45fb496-e3e2-40ec-8a5e-fed01ff00697.png">

is it expected result?

IMO RNT should show only one RN Packager and choose which folder to run
  OR
show all RN Packager and run directly based on folder order without asking which folder to run

ping @SounD120 @RedMickey 🙏

**Comment by jsg2021** [comment]
I would love to be able to fix this as well... I'd also like to be able to mark some projects in the workspace to be ignored by this extension. (as they are linked in libraries, but have RN example apps)

**Comment by EzioLi01** [comment]
Hi @zxccvvv @jsg2021, I have tried on my local but cannot repro this issue. Could you please open an issue with a demo if you still have this issue and need to help? In discussion it's hardly to be found sometimes, it have no count badge in tab. Thanks!
<img width="862" alt="Screenshot 2023-06-15 at 10 47 16" src="https://github.com/microsoft/vscode-react-native/assets/19662219/02ed89b6-ce75-41e6-a686-62d7da6fb071">

**Reply by jsg2021** [reply]
it appears to be fixed in the latest builds. I noticed my duplicate status items reducing to one after the last update

**Reply by EzioLi01** [reply]
Cool, glad to here that!
